### ISSUE TITLE
SKR min E3 v3 Firmware for Ender 3 Max with CR touch

### ISSUE THREAD

**Issue by SteveTheNoob** [bigtreetech/BIGTREETECH-SKR-mini-E3#695]
Hi, I am completely new to this and just replaced my stock ender computer with the SKR after the main board blew up.  Unfortunately all I can find is a precompiled bin file for the regular ender 3 with Bl touch.  IT works fine for my machine but I cant print the full bed size.  My understanding is I have to go into the source code and change three variables related to the bed size and print height and all will be well.  Unfortunately, I know nothing of coding, and cant find the right source code even if I did know what to do with it.  

I have found several sites that tell how to fix the source code and can probably figure it out over the next couple of months but I need the correct source code to use and/or a working compiled version of the code.  Either way I plan on trying to teach myself how to do the coding over the next few months but it sure would be nice to be fully functional while I do it...  LOL

**Comment by radek8** [comment]
The source code is here:
https://github.com/MarlinFirmware/Marlin/tree/bugfix-2.1.x/Marlin

The configuration for comparison is here:
https://github.com/MarlinFirmware/Configurations/tree/bugfix-2.1.x/config/examples/Creality/Ender-3%20Max

https://github.com/MarlinFirmware/Configurations/tree/bugfix-2.1.x/config/examples/Creality/Ender-3/BigTreeTech%20SKR%20Mini%20E3%203.0

**Comment by SteveTheNoob** [comment]
yep that is what I already have found.  none of which is the firmware for the bigtree SKR E3 v3 on an ender 3 Max with CR touch.  It looks like a good starting place when I learn to code but thats a few days away most likely.  🙁


Steven L. Long

SOME PEOPLE ARE LIKE SLINKIES - NOT REALLY GOOD FOR ANYTHING, BUT THEY BRING A SMILE TO YOUR FACE WHEN PUSHED DOWN THE STAIRS.??
________________________________

**Comment by Drkmttr82** [comment]
Im having a similar issue currently and im trying to build my marlin from scratch. I have a sprite pro, btt skr mini3 v3( which is a whole other basket of worms) and a stock lcd. I bought the tft35 but as youll find on these forums not many people have had a worry free time with them. If you come upon any solutions or success let me know and vice versa!

**Comment by haqbany** [comment]
Ready firmware here 

https://youtube.com/shorts/EK_teBw-dPw?feature=share

**Comment by L93camaroz28l** [comment]
> Ready firmware here
> 
> https://youtube.com/shorts/EK_teBw-dPw?feature=share

Yea but that's for the ender 3 pro not the ender 3 max. Wouldn't you have to change build plate volume?

**Comment by haqbany** [comment]
Hi it will work for you the only issue will be the bed size , i had other
users with ender 5 is using the same firmware they reported that os work
for them only the bed is little offside.

U can give it try till you have batter alternative firmware.

**Comment by Rambojoe86** [comment]
I cannot get any firmware to work on my creality ender 3 V2 neo. I installed the SKR mini E3 and built my own cable for the display. Got that working no problem but I cannot get firmware installed I'm ready to throw out the SKR mini and go back to my stock motherboard. I've tried to compile it with no luck whatsoever and I can't find pre-compiled firmware. This is my last cry for help before I toss it out the window and go back to stock. Stock is better than nothing at all cuz my printer has been out for over 2 weeks now and it's got me upset lol

**Comment by haqbany** [comment]
Hi [Rambojoe86],

I understand how frustrating this must be, especially when your printer’s been down for over two weeks. Don’t give up just yet!

Please send me a short video showing exactly what you’re doing when flashing the firmware—how you save the firmware to the SD card, what the folder/file structure looks like, and how you insert and power on the printer. That will help me understand what might be going wrong.

But I’ll be honest—if you’re not patient, and you’re ready to throw the board out, then maybe it’s best to step back for a bit. These upgrades can be tricky at first, but once they’re set up properly, they bring a lot of fun and new possibilities.

Good things come to those who wait—I’ll help you as much as I can if you’re willing to work through it.

Looking forward to the video so we can fix this together!

Best,
RashidSent from my iPhoneOn 6 May 2025, at 1:56 AM, Rambojoe86 ***@***.***> wrote:﻿
Rambojoe86 left a comment (bigtreetech/BIGTREETECH-SKR-mini-E3#695)
I cannot get any firmware to work on my creality ender 3 V2 neo. I installed the SKR mini E3 and built my own cable for the display. Got that working no problem but I cannot get firmware installed I'm ready to throw out the SKR mini and go back to my stock motherboard. I've tried to compile it with no luck whatsoever and I can't find pre-compiled firmware. This is my last cry for help before I toss it out the window and go back to stock. Stock is better than nothing at all cuz my printer has been out for over 2 weeks now and it's got me upset lol

**Comment by Rambojoe86** [comment]
Well being as I am very familiar with modding things such as PlayStations
and dealing with its firmware it's not that I'm a novice in this field. I
am however novice to the 3D printer world. I think it's interesting I was
able to do the harder part of getting the screen to work on the SKR mini
then flashing the firmware. The firmware should be easy. I understand that
you need to keep file paths short and it likes to be named firmware.bin. I
turn the machine off and plug the SD card in with the firmware with the
shortest file path possible and turn the machine on. It does nothing but
show the creality logo. I cannot shoot a video of anything cuz I have since
removed the SKR mini and put the old motherboard back in so I can at least
continue printing until I feel confident I have firmware that will work. I
just find it hard to believe that there is not a pre-compiled firmware for
this machine. Especially because I don't need anything customized I just
need the basic firmware to work.
